.github/workflows/validate-workflows.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate GitHub Actions Workflows
+
+on:
+  pull_request:
+    paths:
+      - "**/*.ya?ml"
+  push:
+    paths:
+      - "**/*.ya?ml"
+  # Scheduled trigger to catch workflow failure resulting from changes to the JSON schema
+  schedule:
```

Comment:
Do we really need a schedule work for this? Can't we just run it on PR basis?

Reply:
addressed